lib/modules/lifecycle.rb

```diff
@@ -46,11 +48,31 @@ def initialize
           entity[:address]['line'] << Faker::Address.secondary_address if (rand < 0.5)
           entity[:city] = location_data['city']
 
-          entity[:med_changes] = Hash.new() { |hsh, key| hsh[key] = [] }
+          #telephone
+          entity[:telephone] = Faker::PhoneNumber.phone_number
+          
+          #birthplace
+          entity[:birth_place] = {
+            'city' => Synthea::Location.selectPoint['city'],
+            'state' => 'MA',
+          }
 
-          choose_socioeconomic_values(entity)
+          #parents
+          mothers_name = Faker::Name.first_name
+          mothers_name = "#{mothers_name}#{(mothers_name.hash % 999)}"
+          mothers_surname = Faker::Name.last_name
+          mothers_surname = "#{mothers_surname}#{(mothers_surname.hash % 999)}"
+          entity[:name_mother] = "#{mothers_name} #{mothers_surname}"
 
-          # TODO update awareness
+          fathers_name = Faker::Name.first_name
+          fathers_name = "#{fathers_name}#{(fathers_name.hash % 999)}"
+          entity[:name_father] = "#{fathers_name} #{entity[:name_last]}"
+
+          #identifiers
+          entity[:identifier_ssn] = "999-#{rand(100..999)}-#{rand(1000..9999)}"
```

Comment:
I purposely used 999 for the first digits, because those are reserved, and so we'd never get a collision with a real SSN.